tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -44,18 +42,33 @@
 from robottelo.decorators import tier2
 from robottelo.decorators import upgrade
 from robottelo.helpers import file_downloader
-from robottelo.rhsso_utils import create_mapper
 from robottelo.rhsso_utils import create_new_rhsso_user
 from robottelo.rhsso_utils import delete_rhsso_user
 from robottelo.rhsso_utils import get_rhsso_client_id
 from robottelo.rhsso_utils import run_command
 
-
 pytestmark = [run_in_one_thread]
 
-
 EXTERNAL_GROUP_NAME = 'foobargroup'
 
+ldap_data = {
```

Comment:
Consider putting these data structures outside of the test module for consistent import and use.  This same dictionary is used within `pytest_fixtures.satellite_auth` for the fixtures.

The fixture could actually return the entity instead of only setting it up, so that these fields can be referenced, or it could return the raw dictionary of values used for setup.

Alternatively these dictionaries could be defined in the fixture module outside of the fixture functions, so that it can be imported. If doing this, make sure that the module can still be read without exception when the settings are missing.

Reply:
I was thinking the same and put the data structure into constants.py but seems like we can not use settings object in constant.py on that time.

Reply:
I wouldn't go as far as putting it in constants, but if its needed in the fixtures and the tests, it would be good to DRY the definition.

Reply:
@mshriver I'm facing issue putting them in constants. if you agree I can put them again in the fixture so that they can be used across all tests. (based on your first comment, let me know if my understanding is correct )